.github/workflows/CI.yaml

```diff
@@ -169,6 +169,19 @@ jobs:
         env:
           RUSTFLAGS: -Cinstrument-coverage
 
+      - name: "Test (regressions): pallets"
+        if: ${{ github.ref != 'refs/heads/master' }}
+        id: test-regressions-pallets
+        continue-on-error: true
```

Comment:
why do we need to continue on errors? Better to save time

Reply:
ok